test/e2e/service.go

```diff
@@ -1145,12 +1145,93 @@ var _ = framework.KubeDescribe("Services", func() {
 		}
 		By("checking kube-proxy health check responses are correct")
 		for n, publicIP := range ips {
-			framework.Logf("Checking health check response for node %s, public IP %s", nodes.Items[n].Name, publicIP)
+			jig.Logf("Checking health check response for node %s, public IP %s", nodes.Items[n].Name, publicIP)
 			// HealthCheck should pass only on the node where num(endpoints) > 0
 			// All other nodes should fail the healthcheck on the service healthCheckNodePort
 			expectedSuccess := nodes.Items[n].Name == readyHostName
 			jig.TestHTTPHealthCheckNodePort(publicIP, healthCheckNodePort, "/healthz", expectedSuccess)
```

Comment:
this request should in theory just get rst and drop immediately, but if the remote end fails to send rst, does it just hang or timeout client side? if so, what's the client timeout?